evalml/automl/automl_search.py

```diff
@@ -653,10 +654,11 @@ def _find_best_pipeline(self):
                 y_train = self.y_train
                 if hasattr(self.data_splitter, "transform_sample"):
                     train_indices = self.data_splitter.transform_sample(X_train, y_train)
-                    X_train = X_train.iloc[train_indices]
-                    y_train = y_train.iloc[train_indices]
+                    X_train = X_train.ww.iloc[train_indices]
+                    y_train = y_train.ww.iloc[train_indices]
```

Comment:
These lines are removed by https://github.com/alteryx/evalml/pull/2253/files right?

Reply:
Yes! Just haven't merged the latest main in yet.